docs/troubleshooting.md

```diff
@@ -116,6 +116,19 @@ be displayed:
 Posit Connect does not allow the content type of a deployment to be changed. If you encounter this
 error, your best option is to create and deploy a new deployment.
 
+## Publisher Output
+
+Publisher output may help to track down problems, it is located in the "Output" window under the "Posit Publisher Deployment" option.
+
+The Output Panel in VSCode or Positron can be opened with ⇧+⌘+U in Mac, Ctrl+⇧+U in Windows.
```

Comment:
Another way to show it which may be a bit easier is doing

`> Output: Show Output Channels...` in the command palette then selecting "Posit Publisher Deployment"

Reply:
Yes, that's helpful! I'll include this option too 👍